src/app/core/services/issue.service.ts

```diff
@@ -152,10 +146,13 @@ export class IssueService {
 
   updateIssueWithComment(issue: Issue, issueComment: IssueComment): Observable<Issue> {
     return this.githubService.updateIssueComment(issueComment).pipe(
-      mergeMap((updatedComment: GithubComment) => {
-        issue.githubComments = [updatedComment, ...issue.githubComments.filter((c) => c.id !== updatedComment.id)];
-        return this.updateIssue(issue);
-      })
+      mergeMap(
+        // eslint-disable-next-line arrow-body-style
+        (updatedComment: GithubComment) => (
+          (issue.githubComments = [updatedComment, ...issue.githubComments.filter((c) => c.id !== updatedComment.id)]),
+          this.updateIssue(issue)
+        )
+      )
```

Comment:
the change here seems to be incorrect - it is changing the logic of this arrow function altogether